Documentation/install/upgrade.rst

```diff
@@ -320,6 +320,37 @@ Annotations:
 
 .. _1.7_required_changes:
 
+1.7.13 Upgrade Notes
+~~~~~~~~~~~~~~~~~~~~
+
+In rare cases, Cilium 1.7.12 or earlier when configured with ENI IPAM mode
+could disrupt connectivity for the node when new ENI devices are created on the
+node. This version fixes the issue and introduces a new flag to handle
+compatibility. This flag is ``--egress-multi-home-ip-rule-compat``.
+
+The flag defaults to ``false`` meaning it will instruct Cilium to **not**
+maintain compatibility and perform a migration to the new ENI datapath, upon
+Cilium upgrade. This operation will modify the routing table to fix a collision
+that in rare cases can cause connectivity loss to the node.
```

Comment:
On second read, `not maintain compatibility` sounds scary if you don't know what's going on. It sounds like there will be network impact. If everything goes well here, there should be no user interaction necessary. I wonder if we can clarify this a bit more.

```suggestion
The flag defaults to ``false`` meaning it will instruct Cilium to perform a
migration to the new ENI datapath upon
Cilium upgrade. This operation will modify the routing table to fix a collision
that in rare cases can cause connectivity loss to the node.
```